backend/compound_library/src/graphql/compound_instances_res.rs

```diff
@@ -0,0 +1,80 @@
+use crate::entities::{compound_instances, compound_types};
+use async_graphql::{ComplexObject, Context, Object};
+use chrono::Utc;
+use opa_client::subject_authorization;
+use sea_orm::{ActiveValue, ColumnTrait, DatabaseConnection, EntityTrait, ModelTrait, QueryFilter};
+use the_paginator::graphql::{CursorInput, ModelConnection};
+use uuid::Uuid;
+
+#[derive(Debug, Clone, Default)]
+pub struct CompoundInstanceQuery;
+
+#[derive(Debug, Clone, Default)]
+pub struct CompoundInstanceMutation;
+
+#[ComplexObject]
+impl compound_instances::Model {
+    async fn compound_types(
+        &self,
+        ctx: &Context<'_>,
+    ) -> async_graphql::Result<Option<compound_types::Model>> {
```

Comment:
`type` may be a more appropriate name as we're already on a `CompoundInstance` node